src/ensembl/src/shared/components/help-popup/HelpPopupButton.tsx

```diff
@@ -28,7 +29,16 @@ import { SlugReference } from './types';
 
 import styles from './HelpPopupButton.scss';
 
-type Props = SlugReference;
+type ClassNames = Partial<{
+  label: string;
+  button: string;
+}>;
```

Comment:
According to the [XD](https://xd.adobe.com/view/56379b1c-f153-4172-a085-6ad119262f49-0364/screen/1be73a31-3fae-42e6-b82f-1f055e464ed5?fullscreen), the button remains the same across all screens of the site, which suggests that there is no need for additional classnames passed through props.

Reply:
Got rid of `ClassNames` type and added only `labelClass` to `props`

Reply:
✅